src/pages/Compose.tsx

```diff
@@ -78,13 +78,11 @@ class Compose extends Component<any, ComposeState> {
   };
 
   onChangeAmount = ({ target: { value } }) => {
-    this.setState({ amount: value && new Number(value) });
+    this.setState({ amount: value && Number(value).toString() }); // TODO: verify if this is correct
```

Comment:
not sure what you're trying to do

Reply:
The type of `amount` is `string` but the type of `new Number(value)` is of course `Number`. Assuming that the previous code is correct, I kept `Number(value)` and invoked the `toString()` method in order to get the proper type and fix the type incompatibility error.

I added the `// TODO` because I'm wondering if `value && Number(value).toString()` is really needed or if just `value` could be enough, considering that according to what I saw in the code the input value should be already a numeric string.